genai-perf/README.md

```diff
@@ -568,14 +572,13 @@ when `--output-tokens-mean` is provided. (default: `0`)
 
 ##### `--random-seed <int>`
 
-The seed used to generate random values. (default: `0`)
```

Comment:
Can we mention the default behavior here?

Reply:
Good call-out. Done.